tests/api.c

```diff
@@ -33265,7 +33265,753 @@ static void test_wolfSSL_EVP_PKEY_assign(void)
+
+    /* Bad cases */
+    AssertIntEQ(wolfSSL_EVP_CIPHER_CTX_set_iv(NULL, iv, ivLen), WOLFSSL_FAILURE);
+    AssertIntEQ(wolfSSL_EVP_CIPHER_CTX_set_iv(ctx, NULL, ivLen), WOLFSSL_FAILURE);
+    AssertIntEQ(wolfSSL_EVP_CIPHER_CTX_set_iv(ctx, iv, 0), WOLFSSL_FAILURE);
+    AssertIntEQ(wolfSSL_EVP_CIPHER_CTX_set_iv(NULL, NULL, 0), WOLFSSL_FAILURE);
+    AssertIntEQ(wolfSSL_EVP_CIPHER_CTX_set_iv(ctx, iv, keyLen), WOLFSSL_FAILURE);
+
+    /* Good case */
+    AssertIntEQ(wolfSSL_EVP_CIPHER_CTX_set_iv(ctx, iv, ivLen), 1);
+
+
+    EVP_CIPHER_CTX_free(ctx);
+    printf(resultFmt, passed);
+#endif
+}
+static void test_wolfSSL_EVP_PKEY_CTX_new_id(void)
+{
+#if defined(OPENSSL_ALL)
+    WOLFSSL_ENGINE* e = 0;
+    int id = 0;
+    EVP_PKEY_CTX *ctx;
+
+    printf(testingFmt, "wolfSSL_EVP_PKEY_CTX_new_id");
+
+    AssertNotNull(ctx = wolfSSL_EVP_PKEY_CTX_new_id(id, e));
+
+    EVP_PKEY_CTX_free(ctx);
+
+    printf(resultFmt, passed);
+#endif
+}
+static void test_wolfSSL_EVP_rc4(void)
+{
+#if defined(OPENSSL_ALL) && !defined(NO_RC4)
+
+    printf(testingFmt, "wolfSSL_EVP_rc4");
+
+    AssertNotNull(wolfSSL_EVP_rc4());
+
+    printf(resultFmt, passed);
+#endif
+}
+static void test_wolfSSL_EVP_enc_null(void)
+{
+#if defined(OPENSSL_ALL)
+
+    printf(testingFmt, "wolfSSL_EVP_enc_null");
+
+    AssertNotNull(wolfSSL_EVP_enc_null());
+
+    printf(resultFmt, passed);
+#endif
+}
+static void test_wolfSSL_EVP_rc2_cbc(void)
+{
+#if defined(OPENSSL_ALL) && defined(WOLFSSL_QT) && !defined(NO_WOLFSSL_STUB)
+
+    printf(testingFmt, "wolfSSL_EVP_rc2_cbc");
+
+    AssertNull(wolfSSL_EVP_rc2_cbc());
+
+    printf(resultFmt, passed);
+#endif
+}
+static void test_wolfSSL_EVP_mdc2(void)
+{
+#if defined(OPENSSL_ALL) && !defined(NO_WOLFSSL_STUB)
+
+    printf(testingFmt, "wolfSSL_EVP_mdc2");
+
+    AssertNull(wolfSSL_EVP_mdc2());
+
+    printf(resultFmt, passed);
+#endif
+}
+static void test_wolfSSL_EVP_md4(void)
+{
+#if defined(OPENSSL_ALL) && !defined(NO_MD4)
+
+    printf(testingFmt, "wolfSSL_EVP_md4");
+
+    AssertNotNull(wolfSSL_EVP_md4());
+
+    printf(resultFmt, passed);
+#endif
+}
+static void test_wolfSSL_EVP_aes_256_gcm(void)
+{
+#if defined(OPENSSL_ALL)
+
+    printf(testingFmt, "wolfSSL_EVP_aes_256_gcm");
+
+    AssertNotNull(wolfSSL_EVP_aes_256_gcm());
+
+    printf(resultFmt, passed);
+#endif
+}
+static void test_wolfSSL_EVP_aes_192_gcm(void)
+{
+#if defined(OPENSSL_ALL)
+
+    printf(testingFmt, "wolfSSL_EVP_aes_192_gcm");
+
+    AssertNotNull(wolfSSL_EVP_aes_192_gcm());
+
+    printf(resultFmt, passed);
+#endif
+}
+static void test_wolfSSL_EVP_ripemd160(void)
+{
+#if defined(OPENSSL_ALL)
+
+    printf(testingFmt, "wolfSSL_EVP_ripemd160");
+
+    AssertNull(wolfSSL_EVP_ripemd160());
+
+    printf(resultFmt, passed);
+#endif
+}
+static void test_wolfSSL_EVP_get_digestbynid(void)
+{
+#if defined(OPENSSL_ALL)
+
+    printf(testingFmt, "wolfSSL_EVP_get_digestbynid");
+
+    AssertNotNull(wolfSSL_EVP_get_digestbynid(4)); /* NID_md5 */
```

Comment:
Let's use the NID enum values here (ex: NID_md5, etc) instead of hard-coded numbers (ex: 4).  Also applies to next two lines.